### ISSUE TITLE
iOS 10 layout issues

### ISSUE THREAD

**Issue by Uragawa** [SvenTiigi/WhatsNewKit#9]
I tried a basic WhatsNewViewController in my app. On the iOS 11 device, it works as expected. On the iOS 10 one, it seems that the order of statements while configuring the labels matters, as setting the font after setting the attributed text overwrites the attributes. Additionally, iOS 10 seems to require estimatedRowHeight to be either set or returned from the delegate for automatic sizing to work properly.

I forked the repo, fixed the issue back in the commit tagged with 1.1.1 (since I personally need Swift 4.1) and did a rebase, should I create a PR for this?

**Comment by SvenTiigi** [comment]
Hey @Uragawa 

Thanks for your detailed bug report 🙌

I've just published a [swift-4.1](https://github.com/SvenTiigi/WhatsNewKit/tree/swift-4.1) Branch and added your iOS 10 Layout fix as well as set the Swift Version to `4.0` to make it compatible up to Swift `4.1`.

You can integrate it with CocoaPods:

```ruby
pod 'WhatsNewKit', :git => 'https://github.com/SvenTiigi/WhatsNewKit.git', :branch => 'swift-4.1'
```

or Carthage:

```ruby
github "SvenTiigi/WhatsNewKit" "swift-4.1"
```

If you found any other bugs you can contribute to the `swift-4.1` branch 👍

Please keep in mind all new features will only be available via the `master` branch which is set to Swift `4.2`

**Comment by Uragawa** [comment]
Woo, thanks for the super quick response!

I updated my Podfile to use the swift-4.1 branch and found that some Swift 4.2 references were left in the source. This is my first time creating a PR so I hope I did it right.

**Comment by SvenTiigi** [comment]
Your [PR](https://github.com/SvenTiigi/WhatsNewKit/pull/10) looks good 👍 and I've merged it to the `swift-4.1` branch.

**Comment by Uragawa** [comment]
Thanks! I'd suggest replicating the layout fix on the master branch (since iOS 10 is still supported as per the podspec), otherwise everything's great now. Since my problem is solved, I'll go ahead and close the issue.
